tests/data/test_sql/moz-fx-data-test-project/test/non_incremental_query_v1/metadata.yaml

```
@@ -7,7 +7,8 @@ labels:
   public_json: true
   incremental: false
   incremental_export: false
-  review_bug: 1999999
+  review_bugs: 
+   - 1999999
```

I think it would make sense to have a test case with multiple review bugs here.
```suggestion
   - 1999999
   - 12121212
```

---

Thank you. I've updated my code with your suggestion.